src/geometric_algebra/nursery/chisolm.lean

```diff
@@ -69,6 +69,28 @@ local infix `*₊`:75 := sym_prod
 
 local infix `*₊ᵥ`:75 := sym_prod_vec
 
+
+def is_orthogonal (a : G₁) (b : G₁) : Prop := sym_prod_vec a b = 0
+
+theorem zero_is_orthogonal (a : G₁) : is_orthogonal 0 a := begin
+  unfold is_orthogonal,
+  unfold sym_prod_vec,
+  unfold prod_vec,
+  simp
```

Comment:
I kind of like to use `squeeze_simp` to replace the output with `simp only [rw_rule1, rw_rule2, ...]` to stabilize the behavior.

Reply:
```suggestion
  simp only [add_zero, zero_mul, mul_zero, add_monoid_hom.map_zero],
```